### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [Jacob-SouthernCity/Jacob-SouthernCity.github.io#3] [Announcements]
# about/

Welcome to Jingze Ma’s Sanctuary!Hi! Don’t know what lead you here, but please feel free to check out my random stuff here. I’m Jingze(Jacob)Ma, 马璟泽 or 小島蓮や(Renya Kojima) in different circumstances. C

https://jacob-southerncity.github.io/about/

**Comment by Chuukanikumann** [comment]
This is outstanding !!! glad to see you doing great ww

**Reply by Jacob-SouthernCity** [reply]
Thanks!

**Comment by dozbear** [comment]
awesome bro!
